cinema_service/settings.py

```diff
@@ -1,40 +1,30 @@
 """
 Django settings for cinema_service project.
-
 Generated by 'django-admin startproject' using Django 4.0.4.
-
 For more information on this file, see
 https://docs.djangoproject.com/en/4.0/topics/settings/
-
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/4.0/ref/settings/
 """
 
 from pathlib import Path
+from datetime import timedelta
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
-
-
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
-
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = (
     "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
 )
-
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
-
 ALLOWED_HOSTS = []
```

Comment:
The `ALLOWED_HOSTS` list is empty. In production, you should specify the allowed hostnames to prevent HTTP Host header attacks.